core/src/main/scala/cromwell/core/io/IoCommandBuilder.scala

```diff
@@ -19,21 +20,14 @@ abstract class PartialIoCommandBuilder {
   def sizeCommand: PartialFunction[Path, Try[IoSizeCommand]] = PartialFunction.empty
   def deleteCommand: PartialFunction[(Path, Boolean), Try[IoDeleteCommand]] = PartialFunction.empty
   def copyCommand: PartialFunction[(Path, Path), Try[IoCopyCommand]] = PartialFunction.empty
-  def hashCommand: PartialFunction[(Path, FileHashStrategy), Try[IoHashCommand]] = PartialFunction.empty
+  def hashCommand: PartialFunction[(Path, FileHashStrategy, IOMetricsCallback), Try[IoHashCommand]] =
+    PartialFunction.empty
   def touchCommand: PartialFunction[Path, Try[IoTouchCommand]] = PartialFunction.empty
   def existsCommand: PartialFunction[Path, Try[IoExistsCommand]] = PartialFunction.empty
   def isDirectoryCommand: PartialFunction[Path, Try[IoIsDirectoryCommand]] = PartialFunction.empty
   def readLinesCommand: PartialFunction[Path, Try[IoReadLinesCommand]] = PartialFunction.empty
 }
 
-object IoCommandBuilder {
-  def apply(partialBuilders: PartialIoCommandBuilder*): IoCommandBuilder =
-    new IoCommandBuilder(partialBuilders.toList)
-
-  def apply: IoCommandBuilder =
-    new IoCommandBuilder(List.empty)
-}
-
```

Comment:
This was only used in the test `StandardFileHashingActorSpec.scala` and it is confusing to have so many entry points to the class.